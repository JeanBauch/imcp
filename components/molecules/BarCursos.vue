<template>
	<div class="flex flex-col justify-center gap-3 md:gap-8">
		<h3 class="text-3xl text-black md:text-[40px] font-semibold mt-9">
			Nossos <span class="text-primary-bordo-500">cursos</span>
		</h3>
		<div
			class="snap-x snap-mandatory overflow-scroll 2xl:overflow-hidden h-52 2xl:h-56 flex justify-between lg:grid lg:grid-cols-6 lg:gap-2 xl:gap-4"
		>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[180px] lg:w-full mr-3"
				:title="'Escola de Aprendizes'"
				:icon="'passarinho'"
				:color="'yellow'"
				:is-nav-bar-cursos="true"
			/>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[180px] lg:w-full mr-3"
				:title="'Piano'"
				:icon="'piano'"
				:color="'orange'"
				:is-nav-bar-cursos="true"
			/>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[180px] lg:w-full mr-3"
				:title="'Regência'"
				:icon="'regencia'"
				:color="'bordo'"
				:is-nav-bar-cursos="true"
			/>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[180px] lg:w-full mr-3"
				:title="'Violino'"
				:icon="'violino'"
				:color="'beige'"
				:is-nav-bar-cursos="true"
			/>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[180px] lg:w-full mr-3"
				:title="'Violão'"
				:icon="'violao'"
				:color="'blue'"
				:is-nav-bar-cursos="true"
			/>
			<ButtonInstrument
				class="snap-center flex-shrink-0 w-[200px] lg:w-full mr-3"
				:title="'Órgão'"
				:icon="'orgao'"
				:color="'yellow'"
				:is-nav-bar-cursos="true"
			/>
		</div>
	</div>
</template>

<script lang="ts">
import Vue from 'vue';
import ButtonInstrument from '~/components/atoms/ButtonInstrument.vue';
export default Vue.extend({
	components: { ButtonInstrument },
});
</script>
