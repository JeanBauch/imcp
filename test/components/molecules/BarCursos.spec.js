import { shallowMount } from '@vue/test-utils';

import BarCursos from '~/components/molecules/BarCursos.vue';

describe('Components:organisms:BarCursos', () => {
	describe('Snapshots', () => {
		it('should render component', () => {
			const wrapper = shallowMount(BarCursos, {});

			expect(wrapper.html()).toMatchSnapshot();
		});
	});
});
