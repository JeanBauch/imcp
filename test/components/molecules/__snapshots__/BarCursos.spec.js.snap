// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components:organisms:BarCursos Snapshots should render component 1`] = `
"<div class=\\"flex flex-col justify-center gap-3 md:gap-8\\">
  <h3 class=\\"text-3xl text-black md:text-[40px] font-semibold mt-9\\">
    Nossos <span class=\\"text-primary-bordo-500\\">cursos</span></h3>
  <div class=\\"snap-x snap-mandatory overflow-scroll 2xl:overflow-hidden h-52 2xl:h-56 flex justify-between lg:grid lg:grid-cols-6 lg:gap-2 xl:gap-4\\">
    <buttoninstrument-stub title=\\"Escola de Aprendizes\\" icon=\\"passarinho\\" color=\\"yellow\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[180px] lg:w-full mr-3\\"></buttoninstrument-stub>
    <buttoninstrument-stub title=\\"Piano\\" icon=\\"piano\\" color=\\"orange\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[180px] lg:w-full mr-3\\"></buttoninstrument-stub>
    <buttoninstrument-stub title=\\"Regência\\" icon=\\"regencia\\" color=\\"bordo\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[180px] lg:w-full mr-3\\"></buttoninstrument-stub>
    <buttoninstrument-stub title=\\"Violino\\" icon=\\"violino\\" color=\\"beige\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[180px] lg:w-full mr-3\\"></buttoninstrument-stub>
    <buttoninstrument-stub title=\\"Violão\\" icon=\\"violao\\" color=\\"blue\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[180px] lg:w-full mr-3\\"></buttoninstrument-stub>
    <buttoninstrument-stub title=\\"Órgão\\" icon=\\"orgao\\" color=\\"yellow\\" isnavbarcursos=\\"true\\" class=\\"snap-center flex-shrink-0 w-[200px] lg:w-full mr-3\\"></buttoninstrument-stub>
  </div>
</div>"
`;
